src/server/handle-api.js

```diff
@@ -5,6 +5,17 @@ import { log } from '../utilities/logger'
 
 let cacheManager = new CacheManager()
 
+const buildBaseUrl = config => {
+  if (config.options && config.options.wordpressDotComHosting) {
```

Comment:
`idx(config, _ => _.options.wordpressDotComHosting)`? I know I'm a pedant.

Reply:
Shit yeah completely forgot that's in Tapestry... is it in Tapestry? Or just MH, let's find out. I assume it is.

Reply:
It certainly is in Tap

Reply:
Consider it pushed.